### DISCUSSION TITLE
locked game size separate to lock window size

### DISCUSSION THREAD

**Discussion by Grant-22** [runelite/runelite#18061] [Ideas]
lately, i have been interested in becoming a better pvmer, and i have found out about optimising the client size.

I used to play on 1750x1370 and now have switched to 1500x900 and have enjoyed the switch with one downside. The screen feels so cluttered, but i have a simple solution 

A separate option to lock my game size at my desired 1500x900, but then lock my window size at 2070x1275(as in picture) or any size desired. This would create a black border around the game screen. Then allow us to move my plugins off the game window, allowing for a clear field of view but also the plugins to still be visible 

At the moment we can make the black border by playing with the game size and interger scaling but this isnt great because then ui isnt scaled to the desired size and the plugin hud isnt able to be dragged into the black border section

![finished](https://github.com/user-attachments/assets/f62a7652-f2e6-4b68-90ed-64ab1fcca668)

This is a quick and dirty mspaint job of what the desired outcome would look like

**Comment by najibed** [comment]
second this!

**Comment by Grant-22** [comment]
It'd also be great for content creation to hide plugins whilst still having them turned on

**Comment by mattthw** [comment]
+1. it would help to remove background clutter while keeping the client the size I want it
